http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -352,12 +352,19 @@ class GceInterpreter[F[_]: Parallel: ContextShift](
           )
         )
         metadata <- getShutdownScript(params.runtimeAndRuntimeConfig.runtime, blocker)
-        _ <- googleComputeService.addInstanceMetadata(
-          params.runtimeAndRuntimeConfig.runtime.googleProject,
-          zoneParam,
-          InstanceName(params.runtimeAndRuntimeConfig.runtime.runtimeName.asString),
-          metadata
-        )
+        _ <- googleComputeService
+          .addInstanceMetadata(
+            params.runtimeAndRuntimeConfig.runtime.googleProject,
+            zoneParam,
+            InstanceName(params.runtimeAndRuntimeConfig.runtime.runtimeName.asString),
+            metadata
+          )
+          .handleErrorWith {
```

Comment:
this is the bug that prevented user from deleting a runtime successfully. Because we clean up error-ed runtime in google during creation, this `addInstanceMetadata` will fail with 404, causing deletion to fail

Reply:
thank you for fixing this!